config/v1/types_image.go

```diff
@@ -36,7 +36,7 @@ type ImageSpec struct {
 	// AdditionalTrustedCA is a reference to a ConfigMap containing additional CAs that
 	// should be trusted during imagestream import.
 	// The namespace for this config map is openshift-config.
-	AdditionalTrustedCA ConfigMapFileReference `json:"additionalTrustedCA,omitempty"`
+	AdditionalTrustedCA ConfigMapNameReference `json:"additionalTrustedCA,omitempty"`
```

Comment:
please document how the content will be used.

Reply:
From the existing godoc:
````
// AdditionalTrustedCA is a reference to a ConfigMap containing additional CAs that
// should be trusted during imagestream import."
```

what else are you looking for?

Reply:
@deads2k bump

Reply:
Uh what key in the config map?

Reply:
all the keys in the configmap.

Reply:
ah `additionalTrustedCA` made me think it was singular.